typescript/cli/src/utils/balances.ts

```diff
@@ -40,18 +40,16 @@ export async function gasBalancesAreSufficient(
   minGas: string,
 ): Promise<boolean> {
   let sufficient = true;
-  await Promise.all(
-    chains.map(async (chain) => {
-      const provider = multiProvider.getProvider(chain);
-      const gasPrice = await provider.getGasPrice();
-      const minBalanceWei = gasPrice.mul(minGas).toString();
-      sufficient = await nativeBalancesAreSufficient(
-        multiProvider,
-        signer,
-        [chain],
-        minBalanceWei,
-      );
-    }),
-  );
+  for (const chain of chains) {
+    const provider = multiProvider.getProvider(chain);
+    const gasPrice = await provider.getGasPrice();
+    const minBalanceWei = gasPrice.mul(minGas).toString();
+    sufficient = await nativeBalancesAreSufficient(
```

Comment:
nativeBalancesAreSufficient() allows a user to continue if any of the chains have insufficient balance. why do we want to let the user continue? It will fail anyways because of insufficient balance

Reply:
because our estimates might be wrong, or a user has topped it up in the meantime.